app/models/supported_ssg.rb

```diff
@@ -113,7 +113,7 @@ def build_latest_map
     end
 
     def cache(key, &block)
-      Rails.cache.fetch("SupportedSsg/datastreams/#{Revision.datastreams}/#{key}", expires_on: 1.day, &block)
+      Rails.cache.fetch("SupportedSsg/datastreams/#{key}", expires_on: 1.day, &block)
```

Comment:
If this expires in a day, how do we ensure that each `key` that is cached is in sync with other keys?

Reply:
As these variables depend on each other, invalidation will happen more or less the same time if it gets expired. The one day expiry is just a safeguard as I did not want to have a perpetual cache that can only be cleared when new content triggers an SSG import. This is only to ensure that if we change something, it prevails without manual intervention from the devprod team. The "standard" way of invalidation is ONLY via the SSG import that clears all three variables. 

I see your point with the programmer's error when adding a new cache key, we could come up with a custom cop for rubocop maybe, but I'd like to do it separately if that's okay.

Reply:
Okay, I have an ugly idea for it without custom cops, give me some time